server/lib/routes/get-metrics-errors.js

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+var logger = require('mozlog')('server.metrics-errors');
+var request = require('request');
+
+var config = require('../configuration');
+var sentryConfig = config.get('sentry');
+
+/**
+ * This module serves as a proxy for front-end errors.
```

Comment:
-1, I'm not sure why we cannot just use endpoints on this server. What are your concerns? I think "proxy" was bad naming on my part. it is an "aggregator" or just "endpoint" just like the other metrics endpoint.